CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+Other improvements:
+- Docs: note `purescript-psa` testing incompatibility
```

Comment:
Hopefully this doesn't imply an incompatibility with `spago test`. Maybe this should be reworded as:
> note `purescript-psa` `stack test` incompatibility